storyblok/storyblok-js-client
# feat: deprecation warning for Richtext

## Pull request type

Jira Link: [INT-](url)




## How to test this PR
1. Build `pnpm run build`
2. Go to `/playground/vanilla`
3. Run `pnpm run dev`
4. Check the console.

## What is the new behavior?



If the user tries to use the `render` method for the `RichTextResolver` the following console warning will appear 
![Screenshot 2024-08-22 at 15 19 12](https://github.com/user-attachments/assets/3473c336-761f-4e78-892a-23f449cde92f)



## Other information
